### PR TITLE
Add Windse

### PR DESCRIPTION
**Pull request by jefalon**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/windse/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  version: {{ fenics_version }}_{{ version }}
+
+source:
+
+  url: https://github.com/NREL/WindSE/raw/{{ branch }}/archives/{{ name }}-{{ fenics_version }}_{{ version }}.tar.gz
+  sha256: 0811c220059867c7d35af6366631005f91c6553c39baa54794a70c9bf59b7665
+
+build:
+  number: 0
+  skip: true  # [win]
```
Comment (xhochy) [inline recipes/windse/meta.yaml]: Can this be `noarch`?  I guess you are skipping Windows as a dependency is missing. That would be ok.

```suggestion
  noarch: python
```

Comment (jefalon) [reply recipes/windse/meta.yaml]: Some of windse's dependencies are not built for windows, so I have to skip building on windows.

Comment (xhochy) [reply recipes/windse/meta.yaml]: As long as the package is building fine on Linux with `noarch`, we can merge with a failing Windows build.

Comment (jefalon) [reply recipes/windse/meta.yaml]: Oh! In that case it should be fine. I originally had `noarch: python` but removed it to skip windows when the windows check failed.

**Comment by jefalon** [comment]
@xhochy Thanks for helping me through this process! I made that commit and, as expected, windows failed to build, but linux and osx built successfully. What do I need to do next?

**Comment by xhochy** [comment]
> @xhochy Thanks for helping me through this process! I made that commit and, as expected, windows failed to build, but linux and osx built successfully. What do I need to do next?

Wait until the feedstock was created, then there should be no Windows build.

**Pull request merged**